### DISCUSSION TITLE
QuickOffice not patched

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1250] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
apparently, it isnt patched if this flag is enabled: [#quickoffice-force-file-download](chrome://flags/#quickoffice-force-file-download) it enables quickoffice editing, which allows it to be used.

**Comment by Sam-hacks1** [comment]
how do you do it?

**Comment by akabutnicer** [comment]
alright tell me when you realize bookmarklets are blocked on versions 115+, i'll wait.

**Reply by schoolexploitkid** [reply]
https://chromewebstore.google.com/detail/injectcode/flhghpihapijancfhnicbnjifdodohpi?hl=en&pli=1

**Reply by akabutnicer** [reply]
😭
